Update teal.Rmd (#759)
insightsengineering/teal: vignettes/teal.Rmd

To learn more about the `teal` framework we recommend first exploring some of the available analysis modules.

> ## Empty `shiny` input cycle

Author: ```suggestion
## Known bugs
```

> - It could be needed to use `options(shiny.useragg = FALSE)` to make the `teal::init` apps more stable, till `ragg` package is not fixed. It is recommented to put this option in the `.Rprofile` file.

Reviewer: IMO it would be worth to put an issue reference so that it not some-unknown-bug but actually we know what we are talking about

Author: just added https://github.com/rstudio/shiny/issues/3704

> - It could be needed to use `options(shiny.useragg = FALSE)` to make the `teal::init` apps more stable, till `ragg` package is not fixed. It is recommented to put this option in the `.Rprofile` file.

Author: ```suggestion
- It could be needed to use `options(shiny.useragg = FALSE)` to make the `teal::init` apps more stable, [till `ragg` package is not fixed](https://github.com/rstudio/shiny/issues/3704). It is recommended to put this option in the `.Rprofile` file.
```

To learn more about the `teal` framework we recommend first exploring some of the available analysis modules.

## Empty `shiny` input cycle

Please read more about the first empty `shiny` input cycle in the `teal::init` apps in the `creating-custom-modules` vignette.

> It could be needed to use `options(shiny.useragg = FALSE)` to make the `teal::init` apps more stable, till `ragg` package is not fixed.

Reviewer 2: Until the `ragg` package is fixed, stability of `teal::init` apps may require the use of `options(shiny.useragg = FALSE)`.

Author: Please make a github suggestion:)
Go to Files Changed click a line or lines and click add Suggestion.

> Please read more about the first empty `shiny` input cycle in the `teal::init` apps in the `creating-custom-modules` vignette.
> It could be needed to use `options(shiny.useragg = FALSE)` to make the `teal::init` apps more stable, till `ragg` package is not fixed.
> It is recommented to put this option in the `.Rprofile` file.

Reviewer 3: ```suggestion
It could be needed to use `options(shiny.useragg = FALSE)` to make the `teal::init` apps more stable, till `ragg` package is not fixed. It is recommented to put this option in the `.Rprofile` file.
Please read more about the first empty `shiny` input cycle in the `teal::init` apps in the `creating-custom-modules` vignette.
```
